extensions/roc-package-web-app-react/src/app/server/reactRenderer.js

```diff
@@ -82,12 +82,21 @@ export function reactRender({
     reduxSagas,
 }) {
     return new Promise((resolve) => {
+        let currentLocation = {};
+
+        history.listen((location) => {
+            currentLocation = location;
+        });
+
         match({ history, routes: createRoutes(store), location: url },
             (error, redirect, renderProps) => {
                 if (redirect) {
-                    log(`Redirect request to ${redirect.pathname + redirect.search}`);
+                    const base = redirect.basename ? redirect.basename : '';
```

Comment:
```js
const base = redirect.basename || '';
```

This would also work and looks a little better in my opinion, not important at all however.